tools/deploy_assisted_installer_configmap.py

```diff
@@ -43,7 +43,7 @@ def main():
 
             versions = {"IMAGE_BUILDER": "installer-image-build",
                         "IGNITION_GENERATE_IMAGE": "assisted-ignition-generator",
-                        "KUBECONFIG_GENERATE_IMAGE": "ignition-manifests-and-kubeconfig-generate",
+                        "KUBECONFIG_GENERATE_IMAGE": "assisted-ignition-generator",
```

Comment:
this line must be removed, it is a leftover from Igal's commit: we are using IGNITION_GENERATE_IMAGE variable, and not KUBECONFIG_GENERATE_IMAGE